pkgs/applications/networking/browsers/offpunk/default.nix

```diff
@@ -0,0 +1,46 @@
+{stdenv, testVersion, makeWrapper, python3, lib, fetchFromGitea, timg, less, offpunk, ...}:
```

Comment:
Afair it is not recommended to use variadic listing.

Reply:
fixed